cmake/custom/tests.cmake

```diff
@@ -1,6 +1,13 @@
+#.rst:
+#
+# autocmake.yml configuration::
+#
+#   docopt: "--enable-tests Enable tests [default: False]."
+#   define: "'-DENABLE_TESTS={0}'.format(arguments['--enable-tests'])"
+
 option(ENABLE_TESTS "Enable test suite" ON)
 
-macro(add_Catch_test _name _labels)
+macro(add_catch_test _name _labels)
```

Comment:
I've rewritten this macro for [PCMSolver](https://github.com/PCMSolver/pcmsolver/blob/release/1.2.Z/cmake/custom/test.cmake) It now only works if you give named arguments, which helps readability:
```
add_Catch_test(
  NAME
    bi_operators_purisima
  LABELS
    bi_operators
    bi_operators_purisima
  REFERENCE_FILES
    uniformdielectric_D_purisima.npy
    vacuum_D_purisima.npy
  )
```
Feel free to copy and adapt at will.

Reply:
This will also be updated in #22